Source/WebKit/Shared/Cocoa/DefaultWebBrowserChecks.mm

```diff
@@ -132,7 +132,7 @@ static bool determineTrackingPreventionStateInternal(bool appWasLinkedOnOrAfter,
     return result != kTCCAccessPreflightDenied;
 }
 
-static RefPtr<WorkQueue>& itpQueue()
+static RefPtr<WorkQueue>& itpQueueSingleton()
```

Comment:
This object isn't really a singleton; we destroy and re-create it.

How about adding

```
static RefPtr<WorkQueue> protectedITPQueue()
{
    return itpQueue();
}
```

and changing call sites to use that?

Reply:
Or you can just do `Ref { itpQueue() }` at the call site.

Reply:
Oh, duh, you're right. Adding protected version makes sense to me.